app/controllers/katello/concerns/organizations_controller_extensions.rb

```diff
@@ -42,9 +43,22 @@ def create
         end
       end
 
+      def update
+        super
+        if params[:content_access_mode] == 'simple_content_access'
+          async_task(::Actions::Katello::Organization::SimpleContentAccess::Enable, params[:id])
+          info "Enabling Simple Content Access for organization #{@taxonomy.name}.", link: { text: "View progress on the Tasks page", href: "/foreman_tasks/tasks" }
```

Comment:
Did you mean to link to the specific task vs the tasks list?

Reply:
It would be nice.  I didn't really look into if I have access to the task ID at that point.  Will investigate & update

Reply:
Updated